#goal definition
string order
---
#result definition
int32[] sequence
---
#feedback
int32[] sequence
